local-antora-playbook.yml

```diff
@@ -6,7 +6,7 @@ site:
 content:
   sources:
     - url: ../kaskada
-      branches: main
+      branches: docs/listing
```

Comment:
you do not need to add this change :) 
The local version of the file is a good seed that people can change for their local development

Reply:
this was added by mistake... I'll remove it